src/app/inventory/d1-stores.ts

```diff
@@ -26,33 +26,39 @@ import { resetItemIndexGenerator } from './store/item-index';
 export function loadStores(): ThunkResult<D1Store[] | undefined> {
   return async (dispatch, getState) => {
     const promise = (async () => {
+      // TODO: check account here
       try {
         let account = currentAccountSelector(getState());
         if (!account) {
           await dispatch(getPlatforms());
           account = currentAccountSelector(getState());
-          if (!account) {
+          if (!account || account.destinyVersion !== 1) {
             return;
           }
         }
+
         resetItemIndexGenerator();
 
         const [defs, , rawStores] = await Promise.all([
           dispatch(getDefinitions()) as any as Promise<D1ManifestDefinitions>,
           dispatch(loadNewItems(account)),
           getStores(account),
         ]);
+
         const lastPlayedDate = findLastPlayedDate(rawStores);
-        const buckets = bucketsSelector(getState())!;
+        const buckets = d1BucketsSelector(getState())!;
 
-        const stores = await Promise.all(
```

Comment:
This turned out not to have been a promise since some long time ago, but we'd never updated to not treat it as a promise.